proto/cacheclient.proto

```diff
@@ -458,7 +473,18 @@ message _ListRemoveRequest {
   }
 }
 
-message _ListRemoveResponse {}
+message _ListRemoveResponse {
+  message _Found {
+    uint32 list_length = 1;
+  }
+
+  message _Missing {}
```

Comment:
this stuff is going to be a little more interesting w/rt the SDKs.  Today, we have several 1.0 SDKs that include a ListRemoveValue function, and they currently have only `Success` and `Failure` response types.

It's easy enough to add the new list length value to the `Success` type, but changing the types be a breaking change.  We could theoretically do something like add a `Miss` response type and leave `Success` in place, mapped to `_Found`, but that would be a little inconsistent with our existing APIs.

@malandis @kvcache I'm interested in thoughts on this.

Reply:
I think it's not a huge issue to expose found/missing through success to the user. In fact, I recommend mapping it to `Success { list_length: 0 }` on the SDK Success object to start out with, and when requested you can add a method `fn was_found() -> bool` (or whatever name you like) to Success without breaking anyone.

Reply:
or I guess `listLength` could be of type `int?` ?

Reply:
You're talking about `momento_sdk::Response::Success::listLength`? You could do that too. Imo it's more annoying than letting it be `int` and having to ask for "did it do the delete?" separately. I'd hazard that the 95% use case is only concerned with `int`. As far as the protos go, I'm just trying to help keep a non-fire-drill avenue open for helping the sdk's serve the remaining vocal 5%, should they speak up.

Reply:
yeah you're right re: 90% case.